### ISSUE TITLE
Broken build due to kind

### ISSUE THREAD

**Issue by johnSchnake** [heptio/sonobuoy#713]
**What steps did you take and what happened:**
Travis is currently failing due to a problem with getting/installing kind.

```
49.72s$ go get sigs.k8s.io/kind
# sigs.k8s.io/kind/pkg/kustomize
../../../sigs.k8s.io/kind/pkg/kustomize/kustomize.go:108:7: undefined: k8sdeps.NewFactory
../../../sigs.k8s.io/kind/pkg/kustomize/kustomize.go:109:26: not enough arguments in call to build.NewCmdBuild
```

**What did you expect to happen:**
Successful build.

**Anything else you would like to add:**
I think they recently went to using gomodules and so we may need to tweak something to properly utilize them. (Could be wrong, but locally I hit that because I was in my GOPATH)

**Comment by johnSchnake** [comment]
https://github.com/kubernetes-sigs/kind/issues/509

**Comment by johnSchnake** [comment]
TL;DR; it seems that we just need to get off from doing `go get ...` and do a clone/checkout of a specific version. With modules and the lack of vendor now `go get` is having problems which aren't really problems with kind; master just isn't guaranteed, only tagged releases.

**Comment by johnSchnake** [comment]
`GO111MODULE=on go get sigs.k8s.io/kind` also works for me locally. Being the fast/easy fix this is sufficient for our our needs in travis IMO.

**Comment by BenTheElder** [comment]
Thanks and sorry about that @johnSchnake, I'd additionally ask to consider using a fixed version so we don't surprise you again :(

0.3 will be out this week, perhaps at that point using a pre-built release binary will be sufficient?

**Comment by johnSchnake** [comment]
Sounds good to me; I had limited availability today so I just did the fastest fix I could find. Happy to fix the version especially if a new release is getting tagged soon.

**Comment by BenTheElder** [comment]
Thanks for understanding and apologies again. 0.3 will indeed land before the end of next week I promise, and should let `go get` without `@master` pick that instead of `HEAD`. I'll send a note when it's out.
